extmod/nimble/nimble/nimble_npl_os.c

```diff
@@ -94,10 +94,10 @@ static void m_free_bluetooth(void *ptr) {
         MP_STATE_PORT(bluetooth_nimble_memory) = NULL;
     }
     m_free(alloc
-    #if MICROPY_MALLOC_USES_ALLOCATED_SIZE
-           , alloc->size
-    #endif
-    );
+        #if MICROPY_MALLOC_USES_ALLOCATED_SIZE
+        , alloc->size
+        #endif
```

Comment:
This is a bit of an odd case- looks like uncrustify bumps up the indent but the fixups don't catch it. Should they?

Reply:
Looking at other examples I think the preference here would be to rewrite this code as follows:

```c
#if MICROPY_MALLOC_USES_ALLOCATED_SIZE
m_free(alloc, alloc->size);
#else
m_free(alloc);
#endif
```